src/main/resources/jenkins/branch/MultiBranchProject/configure-common.jelly

```diff
@@ -23,17 +23,11 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:this="this">
-  <p:config-disableBuild/>
-  <p:config-concurrentBuild />
```

Comment:
Similarly, `concurrentBuild` would make no sense in `MultiBranchProject` even if it existed, because you would not want concurrent branch indexing. You _might_ want to specify that the branch projects use `concurrentBuild`, but if so, this would have to be a `BranchProperty` or the like.